Check/SymfonyVersion.php

```
@@ -103,14 +103,25 @@ function ($value) use ($branch) {
      */
     private function getResponseAndDecode($url)
     {
-        $opts = [
-            'http' => [
-                'method' => 'GET',
-                'header' => "User-Agent: LiipMonitorBundle\r\n",
-            ],
-        ];
-
-        $array = json_decode(file_get_contents($url, false, stream_context_create($opts)), true);
+        if (class_exists(\Symfony\Component\HttpClient\HttpClient::class)) {
+            $client = \Symfony\Component\HttpClient\HttpClient::create();
+            try {
```

Let's remove the try...catch and let the exception bubble up.